Content.Client/Lathe/UI/LatheMenu.xaml.cs

```diff
@@ -71,24 +71,16 @@ public void PopulateMaterials(EntityUid lathe)
             if (!_prototypeManager.TryIndex(materialId, out MaterialPrototype? material))
                 continue;
 
-            var name = Loc.GetString(material.Name);
-            var mat = Loc.GetString("lathe-menu-material-display",
-                ("material", name), ("amount", volume / 100));
-            var volumePerSheet = 0;
-            var maxEjectableSheets = 0;
-
-            if (material.StackEntity != null)
-            {
-                var proto = _prototypeManager.Index<EntityPrototype>(material.StackEntity);
+            var sheetVolume = SheetVolume(material);
+            // rounded here not in locale since doesn't make sense to show 0.5 sheets if you cant eject it
+            var maxEjectableSheets = (int) MathF.Floor((float) volume / sheetVolume);
```

Comment:
don't round it because it shows total volume for all materials. Knowing how much material you have is still useful, even if you can't eject it.

Reply:
you rounded it originally :trollface: